src/cmd/services/m3dbnode/config/limits.go

```diff
@@ -39,4 +41,15 @@ type Limits struct {
 	// process would pause until some of the repaired bytes had been persisted to disk (and subsequently
 	// evicted from memory) at which point it would resume.
 	MaxOutstandingRepairedBytes int64 `yaml:"maxOutstandingRepairedBytes" validate:"min=0"`
+
+	// MaxRecentlyQueriedBlocks controls the max blocks being queried within a given
```

Comment:
nit: `MaxRecentlyQueriedBlocks sets the upper limit on block count within a given...`?